generators/app/templates/CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ series [How to Contribute to an Open Source Project on GitHub][egghead]
 ## Project setup
 
 1. Fork and clone the repo
-2. Run `npm run setup -s` to install dependencies and run validation
+2. Run `npm run setup` to install dependencies and run validation
```

Comment:
I have the `-s` on there because if someone experiences errors they'll get a wall of npm errors that are rarely the problem but that's what people share with you when they report the issue and they don't report the *actual* issue.

Reply:
Now that you mention it I've noticed that issue often myself 😅  I thought this was only for the install script